topic_statistics_demo/include/topic_statistics_demo/imu_talker_listener_nodes.hpp

```diff
@@ -0,0 +1,83 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
FYI the other demos have their respective talkers / listeners in separate files.

Reply:
I followed the [QoS demo](https://github.com/ros2/demos/tree/master/quality_of_service_demo/rclcpp/include/quality_of_service_demo) package's structure of grouping common nodes together.